task/04-arrays-tasks.js

```diff
@@ -146,7 +157,7 @@ function getStringsLength(arr) {
  *    [ 1, 'b', 'c'], 0, 'x'  => [ 'x', 1, 'b', 'c' ]
  */
 function insertItem(arr, item, index) {
-   throw new Error('Not implemented');
+   return arr.splice(index, 0, item); 
```

Comment:
`arr.splice` is not a safe operator. You mutate argument.

Reply:
Tests don't accept this code:  
function insertItem(arr, item, index) {
   let newArr = arr.slice(0, index);
   newArr.push(item, ...arr.slice(index));
   console.log(newArr); //Array(5) [1, 2, 3, 4, 5]
   return newArr; 
}
Maybe tests want to change argument value 'arr'? Because I tried testing without 'return' and with 'splice' and tests accepted it.